pkg/apis/mongodb/v1/mongodb_types.go

```diff
@@ -201,8 +201,11 @@ type AuthMode string
 type MongoDBStatus struct {
 	MongoURI string `json:"mongoUri"`
 	Phase    Phase  `json:"phase"`
-	Members  int    `json:"members"`
-	Message  string `json:"message,omitempty"`
+
+	CurrentStatefulSetReplicas int `json:"currentStatefulSetReplicas"`
+	CurrentReplicaSetMembers   int `json:"currentReplicaSetMembers"`
```

Comment:
Unfortunately there's a name clash between MongoDB ReplicaSet and Kubernetes' Replica Set and this one is actually Mongo's version of it, but not StatefulSet, which correspond to the Kubernetes object.

This will be super confusing at people looking at a status at any given time...

I would definitely vote for:

```golang
	CurrentStatefulSetReplicas int `json:"currentStatefulSetReplicas"`
	CurrentMongoDBMembers   int `json:"currentMongoDBMembers"` 
```

Reply:
I don't have a strong opinion on this but I agree this sounds clearer 👍🏻